dramatiq/cli.py

```diff
@@ -559,24 +561,26 @@ def main(args=None):  # noqa
     )
     log_watcher.start()
 
-    def stop_subprocesses(signum):
+    def stop_subprocesses(signum: signal.Signals):
         nonlocal running
         running = False
 
         for proc in chain(worker_processes, fork_processes):
             try:
-                os.kill(proc.pid, signum)
+                os.kill(proc.pid, signum.value)
             except OSError:  # pragma: no cover
                 if proc.exitcode is None:
-                    logger.warning("Failed to send %r to PID %d.", signum.name, proc.pid)
+                    logger.warning(f"Failed to send {signum.name} to PID {proc.pid}.")
```

Comment:
I would prefer to keep the format strings as they were since f-strings format eagerly, but loggers format lazily.

Reply:
The linter should reject that anyway with [W1203](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html).